tests/network/l2_bridge/test_bridge_nic_hot_plug.py

```
@@ -395,35 +394,47 @@ def hot_unplug_secondary_interface_from_setup(
+def sriov_hot_plug_vm2(namespace, unprivileged_client):
+    with create_vm_for_hot_plug(
         namespace_name=namespace.name,
         vm_name=f"{SRIOV}-{HOT_PLUG_STR}-vm2",
-        sriov_network_for_hot_plug=sriov_network_for_hot_plug,
-        ipv4_address=random_ipv4_address(net_seed=0, host_address=next(index_number)),
         client=unprivileged_client,
+    ) as vm:
+        yield vm
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)